src/Resources/assets/controllers/input-mask_controller.js

```diff
@@ -0,0 +1,27 @@
+import { Controller } from "@hotwired/stimulus"
+import IMask from "imask"
```

Comment:
Consider importing these asynchronously and not synchronously to ensure the land in a separate JavaScript chunk and are only loaded when the controller is connected.

Unless Symfony is doing automatic asynchronous loading of controllers in the background (which I doubt), this is blowing up the JavaScript bundle unnecessarily. A good chunk is under 500 KB.

Reply:
@renestalder I'm not sure if `"fetch": "lazy",` is doing all the magic automatically. I adjusted that for a couple of controllers now.

If this isn't not completely solving the issue please open an issue in the Meta repo. This is out of scope

Reply:
@marcwieland95

> I'm not sure if "fetch": "lazy", is doing all the magic automatically.

Well, I'm neither, but you could look it up, couldn't you? And check the network tab to see if the library is loaded lazily.

At least use `await import(...)` to lazy load the external library on demand inside `connect` if you can't figure out how to change the handling of Symfony here. That's the minimum we can and should do on frontend side.

> This is out of scope

Performance is the minimum we are going to check if we add an external library in addition to minimal security. These things are never out of scope. If we can't reason on the performance impact of adding a library, we are not doing it right.

Reply:
@renestalder Yes, I just checked. I guess `lazy` doesn't include the full file. `imask` was still added to the `app.js` file and wasn't stripped out in a vendor file.

I included now the async import and this does the trick.